doc/sphinx/source/data/dataset-naming-convention.rst

```diff
@@ -3,58 +3,57 @@ NNPDF's dataset naming convention
 =================================
 
 Each dataset implemented in NNPDF must have a unique name, which is a string
-constructed following this [Backus–Naur form]:
+constructed following this [Backus–Naur form]::
 
-```
-<valid dataset name> ::= <experiment> "_" <process>
-                       | <experiment> "_" <process> "_" <energy>
-                       | <experiment> "_" <process> "_" <variant>
-                       | <experiment> "_" <process> "_" <energy> "_" <variant>
+  <valid dataset name> ::= <experiment> "_" <process>
+                        | <experiment> "_" <process> "_" <energy>
+                        | <experiment> "_" <process> "_" <variant>
+                        | <experiment> "_" <process> "_" <energy> "_" <variant>
 
-<experiment> ::= "ATLAS" | "BCDMS" | "CHORUS" | "CMS" | "E605" | "E866"
-               | "E906" | "EMC" | "HERA" | "LHCB" | "NMC" | "NNPDF" | "NUTEV"
+  <experiment> ::= "ATLAS" | "BCDMS" | "CHORUS" | "CMS" | "E605" | "E866"
+                | "E906" | "EMC" | "HERA" | "LHCB" | "NMC" | "NNPDF" | "NUTEV"
```

Comment:
It would be better to drop HERA and add H1 and ZEUS, just as we don't have LHC but ATLAS and CMS

Reply:
@t7phy NO. The inclusive DIS data are for H1 and ZEUS combined, so HERA is better.

Reply:
We should then have also H1 and ZEUS for the DIS+jet data.

Reply:
fine, but we should indeed have H1 and ZEUS also.